EDAM_dev.owl

```diff
@@ -22467,15 +22467,15 @@
     <owl:Class rdf:about="http://edamontology.org/data_2976">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/data_2044"/>
         <created_in>beta12orEarlier</created_in>
-        <documentation rdf:resource="https://en.wikipedia.org/wiki/Protein_primary_structure"/>
+        <rdfs:seeAlso rdf:resource="https://en.wikipedia.org/wiki/Protein_primary_structure"/>
         <oboInOwl:hasDefinition>One or more protein sequences, possibly with associated annotation.</oboInOwl:hasDefinition>
         <oboInOwl:hasExactSynonym>Amino acid sequence</oboInOwl:hasExactSynonym>
         <oboInOwl:hasExactSynonym>Amino acid sequences</oboInOwl:hasExactSynonym>
         <oboInOwl:hasExactSynonym>Protein sequences</oboInOwl:hasExactSynonym>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#data"/>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#edam"/>
         <rdfs:label>Protein sequence</rdfs:label>
-        <rdfs:seeAlso>http://purl.org/biotop/biotop.owl#AminoAcidSequenceInformation</rdfs:seeAlso>
+        <rdfs:seeAlso rdf:resource="http://purl.org/biotop/biotop.owl#AminoAcidSequenceInformation"/>
```

Comment:
Following the temporary "rule" that links to other ontologies stay as literals, this line should be reverted.